README.md

```diff
@@ -24,6 +24,7 @@ The compare tool allows you to use a local config file (`src/config/local.js`) t
 Here, you can customize the following options:
 
 - `mapboxGlAccessToken`: Your Mapbox GL token to allow style reads
+- `config.API_URL`: (optional) Support for a Mapbox flavored style that is served from a different server other than `mapbox.com`, e.g., Mapbox Atlas.
```

Comment:
For our `local.js` config, I think it might be confusing to a user to pass this property this way, especially because we support non-Mapbox maps as well. Could we make this more similar to the `mapboxGlAccessToken` property here and pass the base url as `mapboxBaseApiUrl`?